vllm/engine/arg_utils.py

```diff
@@ -171,6 +174,17 @@ def add_cli_args(
                             choices=['awq', 'squeezellm', None],
                             default=None,
                             help='Method used to quantize the weights')
+        
+        # speculative decoding setting
+        parser.add_argument('--draft-model',
+                            type=str,
+                            default=None,
+                            help='name or path of the huggingface model to use as the draft model')
+        parser.add_argument('--propose-cnt',
+                            type=int,
+                            default=None,
```

Comment:
You may want to add a default value, or you have to assert the input during initialization.